src/strip.rs

```diff
@@ -50,6 +50,21 @@ fn move_until(words: &[&str], it: &mut usize, limit: &str, line: &mut usize) {
     }
 }
 
+fn get_before<'a>(word: &'a str, limit: &str) -> &'a str {
+    let mut it = 0;
+    for c in word.chars() {
+        if limit.contains(c) {
+            break;
+        }
+        it += 1;
+    }
+    if it < word.len() {
+        &word[0..it]
+    } else {
+        word
+    }
+}
```

Comment:
This looks very similar to `word.find(limit_chars).map(|pos| &word[..pos]).unwrap_or(word)` where `limit_chars = ["\t", "\n", etc]`.